Our First Full Feature.md

```diff
@@ -57,22 +57,25 @@ This is a **union type declaration**. It defines a new type that we can use in o
 
 Even though union types and values look similar, they operate in different parts of our program. A union type is just another type, like `String` or `List`, which we can use in our type signatures. Values, on the other hand, can only be used in our implementation code. For instance:
 
+We can use the `case pet of` syntax to **Pattern Match** to branch our code based on which **value** of the **type** we receive.
+
 
 ```elm
-type Pet = Cat | Dog | Rabbit | Turtle
--- this defines the union type `Pet` with four possible values
+type Pet = Cat | Dog | Turtle
+-- this defines the union type `Pet` with three possible values
 
 makeSound : Pet -> String
 --Union types, like `Pet`, can only be used in type signatures.
 makeSound pet =
 --values like `Cat` and `Dog` can't be used in type signatures,
 --but they can be used inside functions.
-  if pet == Cat
-    "meow"
-  else if pet == Dog
-    "woof"
-  else
-    "..silence.."
+  case pet of
+    Cat ->
+      "meow"
+    Dog ->
+      "woof"
```

Comment:
My use of an if statement was intentional. i wanted students to realize they could use values in "normal" implementation code, like a conditional. Case statements can look alien, if you aren't familiar with the syntax, and you could easily assume that the left hand side of a case has to be a type, not a value. 

Students might also think that you *must* use a case statement with a union type, which is of course mistaken.